app/config/AddressLookupConfig.scala

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import play.api.i18n.{Lang, MessagesApi}
+import play.api.libs.json.{JsObject, Json}
+import play.api.mvc.RequestHeader
+import uk.gov.hmrc.hmrcfrontend.config.AccessibilityStatementConfig
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class AddressLookupConfig @Inject()(appConfig: FrontendAppConfig, messagesApi: MessagesApi, accessibilityStatementConfig: AccessibilityStatementConfig){
+
+  def config(continueUrl: String)(implicit request: RequestHeader): JsObject =
+
+    Json.obj (
```

Comment:
you will have to pass English and Welsh resources in some why if you want to prepare this config in the scope this method

Reply:
This file was leftover from a previous attempt, config Json is now created in the connector, I'll add the english and welsh content once it has been confirmed by a BA